MovieApp/MovieApp/App/Features/MovieBrowser/Components/DetailScreen/TitleView/ProgressBarView+Design.swift

```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+extension ProgressBarView: DesignProtocol {
+    
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        circleLayer = CAShapeLayer()
+        layer.addSublayer(circleLayer)
+        
+        progressLayer = CAShapeLayer()
+        layer.addSublayer(progressLayer)
+        
+        percentageLabel = UILabel()
+        addSubview(percentageLabel)
+        
+        userScoreLabel = UILabel()
+        addSubview(userScoreLabel)
+    }
+    
+    func styleViews() {
+        let circularPath = UIBezierPath(arcCenter: CGPoint(x: frame.size.width / 2.0, y: frame.size.height / 2.0), radius: 21, startAngle: CGFloat(-Double.pi / 2), endAngle: CGFloat(3 * Double.pi / 2), clockwise: true)
+        let progressParh = UIBezierPath(arcCenter: CGPoint(x: frame.size.width / 2.0, y: frame.size.height / 2.0), radius: 21, startAngle: CGFloat(-Double.pi / 2), endAngle: CGFloat(endPoint) * 0.06283185307179 + CGFloat(-Double.pi / 2), clockwise: true)
```

Comment:
When using `frame`, please try to do it in layoutSubviews to be sure that the frame is already laid out and has its actual size.

Reply:
[Here](https://github.com/lihter/Movie-App/pull/28/commits/3a1dd8418b7535a8fac5faf7eea2183f097a6eef#diff-cc9e76432cd31ebc1471341e9801dbf4363efceadd2399dae297d7fee3d7bf0fR28-R43)